packages/lit-dev-content/site/tutorials/content/reactivity/04.md

```diff
@@ -0,0 +1,9 @@
+One way your component can react to property changes is to compute values based on the new and existing property values.
+
+If you need to compute a value to render, you can compute it in the `render()` method itself. However a more general solution is to compute values in the `willUpdate()` callback. This has a couple of advantages:
+
+-   You can make computed properties available to [reactive controllers](/docs/composition/controllers/). The `willUpdate()` callback is called before the `hostUpdate()` callback is called on any [reactive controllers](/docs/composition/controllers/).
```

Comment:
I think this list could include something along the line that it's a good spot to reconcile between reactive properties and won't trigger rerender unlike in the render function (right?)

Reply:
Rewrote this section per Steve's input. Apparently only changes in `updated()` trigger rerender.